engine/src/main/java/org/terasology/logic/characters/CharacterMovementComponent.java

```diff
@@ -71,6 +71,8 @@
     // Distance since last footstep
     public float footstepDelta;
 
+    // Distance desired from ground
+    public float groundDistance;
```

Comment:
I must have missed the conversation part about what this is about... what is this for?